chapters/my_first_jina_app/index.md

```diff
@@ -0,0 +1,389 @@
+# Build your first Jina app
+
+## 👋 Introduction
+
+This tutorial guides you through building your own neural search app using the [Jina framework](https://github.com/jina-ai/jina/). 
+
+![](./images/jinabox-wikipedia.gif)
+
+Our example program will be a simple neural search engine for text. It will take a user's input, and return a list of sentences from Wikipedia that match most closely.
```

Comment:
It will take a user's input and return a list of Wikipedia sentences that match most closely.